src/app/resources-page/resources-page.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Http } from '@angular/http';
+
+const _resources = ['Development', 'Education', 'Community'];
+const URL_PATH = '/resources/resources.json';
+
+type ResourceItem = {
+  desc: String,
+  rev: Boolean,
+  title: String,
+  url: String
+}
+
+type ResourceGroup = {
+  key: String,
+  items: any[]
+}
+
+@Component({
+  selector: 'resources-page',
+  templateUrl: 'resources-page.component.html',
+  styleUrls: ['resources-page.component.css']
+})
+export class ResourcesPageComponent implements OnInit {
+  developmentItems = [];
+  educationItems = [];
+  communityItems = [];
+
+  adjustJumpnav: boolean = false;
+
+  constructor(private http: Http) {
+    http.get(URL_PATH)
+      .toPromise()
+      .then(resp => {
+        const result = resp.json();
+        _resources.forEach((k, i) => {
+          const key = `${k.toLowerCase()}Items`;
+          this[key] = flattenFirebaseObject(result[k]);
+        });
+      })
+      .catch(e => e);
+  }
+
+  changeScrollPos() {
+    if (window.scrollY > 250) {
+      this.adjustJumpnav = true;
+    } else {
+      this.adjustJumpnav = false;
+    }
+  }
+
+  ngOnInit() {
+    window.addEventListener('scroll', this.changeScrollPos.bind(this), false);
+  }
+
+  ngOnDestroy() {
+    window.removeEventListener('scroll', this.changeScrollPos.bind(this), false);
+  }
+}
+
+function flattenFirebaseObject(result): ResourceGroup[] {
```

Comment:
Before we were leveraging the Firebase SDK to extract the data (and a pipe to parse the data). Rather than doing that, I simply fetch the firebase json file on npm postinstall, and flatten the file to conform to `ResourceGroup[]`.